src/utils/signals.c

```diff
@@ -0,0 +1,14 @@
+#include "minishell.h"
+
+void signals_handler(int sig) {
+	// TODO kill a running process
+
+	// This clears the readline input
+	if(sig == SIGINT) {
```

Comment:
should we have a function specifically for each signal like 
```c
void sigint_handler(int sig)
{}
```
or `void ctrl_c()`
And avoid the if condition

Reply:
ctrl d is already handled, so we will only have to handle ctrl_c in two different ways, depending if a process is running or we are reading a line

Reply:
But agree with the naming!